sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Metrics/Metrics.cs

```diff
@@ -78,6 +81,23 @@ protected override bool IsCommentTrivia(SyntaxTrivia trivia)
             }
         }
 
+        protected override bool IsDocumentationCommentTrivia(SyntaxTrivia trivia)
+        {
+            switch (trivia.Kind())
+            {
+                case SyntaxKind.SingleLineCommentTrivia:
+                case SyntaxKind.MultiLineCommentTrivia:
+                    return trivia.ToString().TrimStart().StartsWith("///");

```

Comment:
Roslyn does not recognize the C# "documentation" comment trivia as such unless you specify `"/p:DocumentationFile=foo.xml"` as command line argument